src/components/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Sidebar from './Sidebar';
+// import blankHead from '';
+type Profileprops = {
+    image?: string;
+    name?: string;
+    role?: string;
+};
+
+function Dashboard({ image, name, role }: Profileprops) {
+  return (
+    <div className="flex flex-col w-[15rem]">
+      <div className="bg-cyan-500 flex flex-row items-center px-8 py-2 gap-2 h-[10%] mt-20">
+        <img src={image} alt="profile" className="w-[50px]" />
```

Comment:
Please use an icon, for now. Please if you check in the preview, the image is not even being imported / rendered correctly. If you must use an image, at least trying an external link.